components/brave_wallet/renderer/brave_wallet_js_handler.cc

```diff
@@ -18,6 +18,7 @@
 #include "brave/components/brave_wallet/common/hex_utils.h"
```

Comment:
As a follow up pls move this file to ethereum_js_handler.cc or similar.

Reply:
https://github.com/brave/brave-browser/issues/21944